sites/broadbandtechreport/server/components/inquiry-form.marko

```diff
@@ -0,0 +1,12 @@
+import { getAsObject } from '@base-cms/object-path';
+
+$ const content = getAsObject(input, 'content');
+
+<div class="inquiry-form">
+  <div class="inquiry-form-header">Request More Information</div>
+  <div class="inquiry-form-body">
+    <p>Fill out the form below to request more information about <marko-web-content-name tag=null obj=content />.</p>
+    <hr>
+    <marko-web-browser-component name="InquiryForm" props={ contentId: content.id, content: content }/>
```

Comment:
You'll need to register the Inquiry browser component here:
https://github.com/base-cms-websites/endeavor-business-media/blob/master/sites/broadbandtechreport/browser/index.js

See https://github.com/base-cms-websites/endeavor-business-media/blob/master/sites/laserfocusworld/browser/index.js for an example

Reply:
done